### PR TITLE
SLI-1204 Fix for SSF-533

### PR DESCRIPTION
**Pull request by nquinquenel**

### CODE REVIEW

File: src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt [hunk 1]
```diff
@@ -321,8 +324,25 @@ object SonarLintIntelliJClient : SonarLintClient {
     override fun assistCreatingConnection(params: AssistCreatingConnectionParams): CompletableFuture<AssistCreatingConnectionResponse> {
         return CompletableFuture.supplyAsync {
             val serverUrl = params.serverUrl
-            val message = "No connections configured to '$serverUrl'."
-            if (!showConfirmModal(OPENING_FINDING_TITLE, message, "Create connection", null)) {
+            val warningTitle = "Do you trust this SonarQube server?"
```
Comment (eray-felek-sonarsource) [inline src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt:327]: We could move this constant to companion object as WARNING_TITLE

File: src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt [hunk 2]
```diff
@@ -321,8 +324,25 @@ object SonarLintIntelliJClient : SonarLintClient {
...
+            val message = """
+                        The server <b>${escapeHtml(serverUrl)}</b> is attempting to set up a connection with SonarLint. Letting SonarLint connect to an untrusted SonarQube server is potentially dangerous.
+                        
+                        If you don’t trust this server, we recommend canceling this action and <a href="$CONNECTED_MODE_SETUP_LINK">manually setting up Connected Mode<icon src="AllIcons.Ide.External_link_arrow" href="$CONNECTED_MODE_SETUP_LINK"></a>.
+                    """.trimIndent()
+            val connectButtonText = "Connect to this SonarQube server"
```
Comment (eray-felek-sonarsource) [inline src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt:333]: same

File: src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt [hunk 3]
```diff
@@ -321,8 +324,25 @@ object SonarLintIntelliJClient : SonarLintClient {
...
+            val dontTrustButtonText = "I don't trust this server"
```
Comment (eray-felek-sonarsource) [inline src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt:334]: same

File: src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt [hunk 4]
```diff
@@ -321,8 +324,25 @@ object SonarLintIntelliJClient : SonarLintClient {
     override fun assistCreatingConnection(params: AssistCreatingConnectionParams): CompletableFuture<AssistCreatingConnectionResponse> {
         return CompletableFuture.supplyAsync {
             val serverUrl = params.serverUrl
-            val message = "No connections configured to '$serverUrl'."
-            if (!showConfirmModal(OPENING_FINDING_TITLE, message, "Create connection", null)) {
+            val warningTitle = "Do you trust this SonarQube server?"
+            val message = """
```
Comment (eray-felek-sonarsource) [inline src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt:328]: same

File: src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt (hunk 1 above)
Comment (nquinquenel) [reply src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt:327]: This is only used in this method so I'd lean towards keeping the variable in the method where it is used, WDYT?

**Review by eray-felek-sonarsource — APPROVED** [review]

**Pull request merged**